app/qml/FeaturePanel.qml

```diff
@@ -175,8 +175,9 @@ Drawer {
                     property color backgroundColorInactive: "grey"
                     property color fontColor: InputStyle.fontColor
                     property color activeColor: InputStyle.fontColor
+                    property color attentionColor: "red"
```

Comment:
Can we have a bit darker color so it better matches with the other dark green buttons... e.g. #aa0000 ?